sql/src/main/java/io/crate/analyze/relations/JoinPairs.java

```diff
@@ -81,7 +81,8 @@ public static JoinPair fuzzyFindPair(List<JoinPair> pairs, QualifiedName lhs, Qu
             ListIterator<JoinPair> it = pairs.listIterator();
             while (it.hasNext()) {
                 JoinPair pair = it.next();
-                if (pair.joinType() != JoinType.SEMI && pair.equalsNames(rhs, lhs)) {
+                JoinType joinType = pair.joinType();
+                if (joinType != JoinType.SEMI && joinType != JoinType.ANTI && pair.equalsNames(rhs, lhs)) {
```

Comment:
maybe we should add a method to the joinType that can be used to check if it supports reverse?